docs/setup/clouds/index.md

```diff
@@ -31,7 +31,34 @@ The currently supported cloud providers are:
 ## Supporting a new cloud provider
 
 To develop a new cloud provider add a new folder in
-`srcs/runners_manager/vm_creation` containing a `CloudManager`
-object matching [CloudManager] interface.
+`srcs/runners_manager/vm_creation` containing a
+`CloudManager` object matching [CloudManager] interface.
+
+### Retrieving VM parameters
+
+To communicate with the application, the user will provide
+a lot of information in a `yml` configuration file, such as
+`image_id`, `instance_type`, `security_group_ids`, and so on.
```

Comment:
Be less precise, as those names are specific to AWS. Per example, image, flavor, security settings, etc.